src/LTOCli/Commands/Sponsorship.py

```diff
@@ -2,32 +2,61 @@
 from LTO.Transactions.Sponsorship import Sponsorship
 from LTO.Transactions.CancelSponsorship import CancelSponsorship
 from LTOCli import Config
+from LTO.PublicNode import PublicNode
 
 def func(nameSpace,parser):
-    if nameSpace.option[0] == 'create':
+    if vars(nameSpace)['subparser-name-sponsorship'] == 'create':
```

Comment:
Again, I believe switch statement would be more suitable

Reply:
python doesn't have a switch, it has to be implemented and I think the current available implementations are rather ugly

Reply:
There is, since Python 3.10, however, I don't know which python 3 version we aim to support.